lib/middleware/request_tracker.rb

```diff
@@ -198,13 +266,63 @@ def self.get_data(env, result, timing, request = nil)
     h
   end
 
+  def self.extract_topic_params_deferred_track(env, request)
```

Comment:
Not totally happy with these 3 methods laying around here...open to alternatives